src/components/Activity/TimeTag.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { GU, textStyle, useTheme } from '@aragon/ui'
+import dayjs from 'dayjs'
+import relativeTime from 'dayjs/plugin/relativeTime'
+import useNow from '../../hooks/useNow'
+
+dayjs.extend(relativeTime)
+
+function getRelativeTime(now, targetDate) {
+  return dayjs(targetDate)
+    .from(now)
+    .replace(/minutes?/, 'min')
+    .replace(/seconds?/, 'sec')
+    .trim()
+}
+
+function TimeTag({ date, label, ...props }) {
+  const theme = useTheme()
+  const now = useNow()
+  const targetDate = new Date(date)
```

Comment:
should we use dayjs here? asking because sometimes we use javascript Date and some others Dayjs

Reply:
Yes! I also consolidated all the date related things into date-utils: https://github.com/aragon/court-dashboard/pull/100/commits/a54e3f523631e768c866da787b8ccc42230b3143